### PR TITLE
docs(35/start) provide default ports for enterprise

### PR DESCRIPTION
**Pull request by h3xar0n**
### Summary

In the Getting Started guide, once a user has started Kong, they have a list of default ports used with Kong Enterprise. 

Each port listing will also link to the related configuration property in case the organization wants to know where to change the ports used or to deactivate unused ones. 

### Checklist:

- [x] [Commit message & atomicity](https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md#commit-atomicity) checked
- [x] Documentation 
- [x] [Adheres to Kong style guide](https://github.com/Kong/docs.konghq.com/blob/master/STYLEGUIDE.md)
- [x] Spellchecked my updates
- [x] Tagged "Team Docs" as reviewers
- [x] Ready to be merged 

For this PR:
- [x] Update entity names with SG
- [x] Check accuracy of ports (some may have changed since 0.31)
- [x] Link to Configuration Property Reference

### CODE REVIEW

File: app/enterprise/0.35-x/getting-started/start-kong.md [hunk 1]
```diff
@@ -95,11 +96,26 @@ allowing you to point to [your own configuration](/1.0.x/configuration/#configur
...
 To test that Kong Enterprise has successfully started with a **Super Admin**, 
-visit Kong Manager's URL. By default, it is `http://localhost:8002`. 
+visit Kong Manager's URL. By [default](#default-ports), it is on port `:8002`. 
 
 The username is `kong_admin` and the password is the one set in 
 [Step 1](#step-1).
 
+### Default Ports
+
+By default Kong listens on the following ports:
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/start-kong.md]:
```suggestion
By default, Kong Enterprise listens on the following ports:
```

File: app/enterprise/0.35-x/getting-started/start-kong.md [hunk 2]
```diff
@@ -95,11 +96,26 @@ allowing you to point to [your own configuration](/1.0.x/configuration/#configur
...
+
+- `:8000`: incoming HTTP traffic from Consumers, and forwarded to Upstream
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/start-kong.md]:
```suggestion
- `:8000`: incoming HTTP traffic from **Consumers**, and forwarded to upstream 
```

File: app/enterprise/0.35-x/getting-started/start-kong.md [hunk 3]
```diff
@@ -95,11 +96,26 @@ allowing you to point to [your own configuration](/1.0.x/configuration/#configur
...
 
 The username is `kong_admin` and the password is the one set in 
 [Step 1](#step-1).
 
+### Default Ports
+
+By default Kong listens on the following ports:
+
+- `:8000`: incoming HTTP traffic from Consumers, and forwarded to Upstream 
+  Services.
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/start-kong.md]:
```suggestion
  **Services**.
```

File: app/enterprise/0.35-x/getting-started/start-kong.md [hunk 4]
```diff
@@ -95,11 +96,26 @@ allowing you to point to [your own configuration](/1.0.x/configuration/#configur
...
+- `:8443`: incoming HTTPS traffic. This port behaves similarly to the `:8000` 
+  port, except that it expects HTTPS traffic only. 
+- `:8003`: Dev Portal traffic, assuming the Dev Portal is enabled.
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/start-kong.md]:
```suggestion
- `:8003`: Dev Portal traffic, assuming the Dev Portal is **enabled**.
```

File: app/enterprise/0.35-x/getting-started/start-kong.md [hunk 5]
```diff
@@ -95,11 +96,26 @@ allowing you to point to [your own configuration](/1.0.x/configuration/#configur
...
+- `:8004`: Dev Portal `/files` traffic, assuming the Dev Portal is enabled.
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/start-kong.md]:
```suggestion
- `:8004`: Dev Portal **`/files` traffic**, assuming the Dev Portal is **enabled**.
```

Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/start-kong.md]:
```suggestion
- `:8004`: Dev Portal `/files` traffic, assuming the Dev Portal is **enabled**.
```

File: app/enterprise/0.35-x/getting-started/start-kong.md [hunk 6]
```diff
@@ -95,15 +96,30 @@ allowing you to point to [your own configuration](/1.0.x/configuration/#configur
...
+visit Kong Manager's URL. By [default](#default-ports), it is on port `:8002`. 
 
 The username is `kong_admin` and the password is the one set in 
 [Step 1](#step-1).
 
+### Default Ports
+
+By default, Kong Enterprise listens on the following ports:
+
+- `:8000`: incoming HTTP traffic from **Consumers**, and forwarded to upstream
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/start-kong.md]:
```suggestion
- [`:8000`](/enterprise/{{page.kong_version}}/property-reference/#proxy_listen): incoming HTTP traffic from **Consumers**, and forwarded to upstream 
```

File: app/enterprise/0.35-x/getting-started/start-kong.md [hunk 7]
```diff
@@ -95,15 +96,30 @@ allowing you to point to [your own configuration](/1.0.x/configuration/#configur
...
 The username is `kong_admin` and the password is the one set in 
 [Step 1](#step-1).
 
+### Default Ports
+
+By default, Kong Enterprise listens on the following ports:
+
+- `:8000`: incoming HTTP traffic from **Consumers**, and forwarded to upstream 
+  **Services**.
+- `:8443`: incoming HTTPS traffic. This port behaves similarly to the `:8000`
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/start-kong.md]:
```suggestion
- [`:8443`]((/enterprise/{{page.kong_version}}/property-reference/#proxy_listen)): incoming HTTPS traffic. This port behaves similarly to the `:8000` 
```

File: app/enterprise/0.35-x/getting-started/start-kong.md [hunk 8]
```diff
@@ -95,15 +96,30 @@ allowing you to point to [your own configuration](/1.0.x/configuration/#configur
...
 
+### Default Ports
+
+By default, Kong Enterprise listens on the following ports:
+
+- `:8000`: incoming HTTP traffic from **Consumers**, and forwarded to upstream 
+  **Services**.
+- `:8443`: incoming HTTPS traffic. This port behaves similarly to the `:8000` 
+  port, except that it expects HTTPS traffic only. 
+- `:8003`: Dev Portal traffic, assuming the Dev Portal is **enabled**.
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/start-kong.md]:
```suggestion
- [`:8003`](/enterprise/{{page.kong_version}}/property-reference/#portal_gui_listen): Dev Portal traffic, assuming the Dev Portal is **enabled**.
```

File: app/enterprise/0.35-x/getting-started/start-kong.md [hunk 9]
```diff
@@ -95,15 +96,30 @@ allowing you to point to [your own configuration](/1.0.x/configuration/#configur
...
+- `:8004`: Dev Portal **`/files` traffic**, assuming the Dev Portal is **enabled**.
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/start-kong.md]:
```suggestion
- [`:8004`](/enterprise/{{page.kong_version}}/property-reference/#portal_api_listen): Dev Portal **`/files`** traffic over HTTP, assuming the Dev Portal is **enabled**.
- [`:8447`](/enterprise/{{page.kong_version}}/property-reference/#portal_api_listen): Dev Portal **`/files`** traffic over HTTPS, assuming the Dev Portal is **enabled**.
```

File: app/enterprise/0.35-x/getting-started/start-kong.md [hunk 10]
```diff
@@ -95,15 +96,30 @@ allowing you to point to [your own configuration](/1.0.x/configuration/#configur
...
+- `:8001`: Admin API listens for HTTP traffic.
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/start-kong.md]:
```suggestion
- [`:8001`](/enterprise/{{page.kong_version}}/property-reference/#admin_api_uri): Admin API listens for HTTP traffic.
```

File: app/enterprise/0.35-x/getting-started/start-kong.md [hunk 11]
```diff
@@ -95,15 +96,30 @@ allowing you to point to [your own configuration](/1.0.x/configuration/#configur
...
+- `:8444`: Admin API listens for HTTPS traffic.
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/start-kong.md]:
```suggestion
- [`:8444`](/enterprise/{{page.kong_version}}/property-reference/#admin_api_uri): Admin API listens for HTTPS traffic.
```

File: app/enterprise/0.35-x/getting-started/start-kong.md [hunk 12]
```diff
@@ -95,15 +96,30 @@ allowing you to point to [your own configuration](/1.0.x/configuration/#configur
...
+- `:8002`: Kong Manager listens for HTTP traffic.
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/start-kong.md]:
```suggestion
- [`:8002`](/enterprise/{{page.kong_version}}/property-reference/#admin_gui_listen): Kong Manager listens for HTTP traffic.
```

File: app/enterprise/0.35-x/getting-started/start-kong.md [hunk 13]
```diff
@@ -95,15 +96,30 @@ allowing you to point to [your own configuration](/1.0.x/configuration/#configur
...
+- `:8445`: Kong Manager listens for HTTPS traffic.
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/getting-started/start-kong.md]:
```suggestion
- [`:8445`](/enterprise/{{page.kong_version}}/property-reference/#admin_gui_listen): Kong Manager listens for HTTPS traffic.
```

**Comment by levfishbluefish** [comment]
I am worried the location is a bit buried, but its OK for now, will put it in user testing

File: app/enterprise/0.35-x/getting-started/start-kong.md [hunk 14]
```diff
@@ -95,15 +96,32 @@ allowing you to point to [your own configuration](/1.0.x/configuration/#configur
...
 The username is `kong_admin` and the password is the one set in 
 [Step 1](#step-1).
 
+### Default Ports
+
+By default, Kong Enterprise listens on the following ports:
+
+- [`:8000`](/enterprise/{{page.kong_version}}/property-reference/#proxy_listen): incoming HTTP traffic from **Consumers**, and forwarded to upstream 
+  **Services**.
+- [`:8443`](/enterprise/{{page.kong_version}}/property-reference/#proxy_listen)): incoming HTTPS traffic. This port behaves similarly to the `:8000`
```
Comment (levfishbluefish) [inline app/enterprise/0.35-x/getting-started/start-kong.md]:
```suggestion
- [`:8443`](/enterprise/{{page.kong_version}}/property-reference/#proxy_listen): incoming HTTPS traffic. This port behaves similarly to the `:8000` 
```

**Comment by h3xar0n** [comment]
@levfishbluefish true, it's buried, and the context where it comes up is simply the first time we mention a default port. It would more logically go in the Deployment section after the installation guides, since default/required ports are probably matters that an org wants to configure before even starting Kong.

Alas, user testing will let us know for sure.

**Pull request merged**